geomesa-hbase/geomesa-bigtable-tools/bin/geomesa-bigtable

```diff
@@ -24,15 +24,7 @@ if [[ -n "$HADOOP_HOME" ]]; then
 fi
 
 # Warn user about missing environment variables
-if [[ -z "$HADOOP_HOME" ]]; then
-  message="Warning: you have not set HADOOP_HOME as an environment variable."
-  message="${message}\nGeoMesa tools will not run without the appropriate Hadoop jars in the tools classpath."
-  message="${message}\nPlease ensure that those jars are present in the classpath by running 'geomesa-bigtable classpath' ."
-  message="${message}\nTo take corrective action, please place the necessary jar files in the lib directory of geomesa-tools."
-  echo >&2 ""
-  echo -e >&2 "$message"
-  echo >&2 ""
-elif [[ -z "$HADOOP_CONF_DIR" ]]; then
+if [[ -z "$HADOOP_CONF_DIR" ]]; then
```

Comment:
technically I think this won't check if its empty, right? It may be useful to use the ```[[ -z "{VAR+x}" ]]``` construct in bash like this:

```
ABC="" bash -c 'if [[ -z "$ABC" ]]; then echo "set"; else echo "empty"; fi '
set
ABC="" bash -c 'if [[ -z "${ABC+x}" ]]; then echo "set"; else echo "empty"; fi '
empty
```

we probably have a lot of plain ```-z``` when we actually want to check two things (is it set and is it non empty) we may want to clean them up at some point.